### ISSUE TITLE
[Informative] Observable changes because of Unicode 12

### ISSUE THREAD

**Issue by mathiasbynens** [tc39/ecma262#1467]
Output of [`node compare.js`]( https://gist.github.com/mathiasbynens/c08f45efdcdb9389adf3da57e16683de):

```
$ node compare.js 
Comparing Unicode v12.0.0 to Unicode v11.0.0…
Binary_Property: 0 new values
General_Category: 0 new values
Script: 4 new values
- Elymaic (Elym)
- Nandinagari (Nand)
- Nyiakeng_Puachue_Hmong (Hmnp)
- Wancho (Wcho)
Script_Extensions: 4 new values
- Elymaic (Elym)
- Nandinagari (Nand)
- Nyiakeng_Puachue_Hmong (Hmnp)
- Wancho (Wcho)
Binary_Property/ID_Start: 0 removals
Binary_Property/ID_Start: 224 new code points
Binary_Property/ID_Continue: 0 removals
Binary_Property/ID_Continue: 279 new code points
General_Category/Space_Separator: 0 removals
General_Category/Space_Separator: 0 new code points
Case_Folding=S: 0 removals
Case_Folding=S: 0 new code points
Case_Folding=F: 0 removals
Case_Folding=F: 0 new code points
```

TL;DR identifier symbols are expanded and some new properties and aliases are supported in RegExp Unicode property escapes.
